config/test.js

```diff
@@ -10,4 +10,8 @@ module.exports = {
   enableRequestID: false,
 
   mozillaUserId: 1337,
+  
+  // Don't run with any active experiments, otherwise tests try to send an
+  // enrollment event.
```

Comment:
"because tests run on the server by default and experiments are client-side only."?